frontend/src/app/libs/common-components/components/event-tile/event-tile.component.html

```diff
@@ -0,0 +1,55 @@
+<section class="app-event-tile" 
+		[class.app-event-tile--draft]="isDraft">
+		
+	<div class="app-event-tile__container">
+		<h3 class="app-event-tile__title" [attr.title]="name"> {{ name }} </h3>
```

Comment:
Suppose that we can use just [title]

Reply:
no need to add title for now, design does not contain such functionality